.gitlab/benchmarks.yml

```diff
@@ -0,0 +1,33 @@
+variables:
+  BASE_CI_IMAGE: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/benchmarking-platform:dd-trace-php-82-dev
```

Comment:
Where and how is this image built?

Reply:
I added an inline comment so we will know in the future.
https://github.com/DataDog/benchmarking-platform/tree/dd-trace-php